cspell.config.js

```diff
@@ -70,7 +70,7 @@ const config = {
         'src/*',
 
         // Ignore release notes for now.
-        'docs/release-notes.md',
+        'general/releases/*',
```

Comment:
When (issue number) these pages will stop being ignored? Maybe a separate issue should be raise to not forget about that ;-)

Reply:
I'm not planning to fix legacy ones at this time.
generally speaking, release notes incldue the issue description as-is from most issues so we're gonna see a lot of cspell violations. We want to think about how we handle that generally though.